runtime-main-mirror.yml

```diff
@@ -1,5 +1,16 @@
-# Disable triggers as it should only be called using Maestro
-trigger: none
+resources:
```

Comment:
I'm trying to trigger it from https://dev.azure.com/dnceng/internal/_git/dotnet-runtime?path=/&version=GBmain&_a=history but I'm not sure about the syntax

Reply:
I think this looks correct.